### DISCUSSION TITLE
using rdf4j-server sparql endpoint

### DISCUSSION THREAD

**Discussion by naturzukunft** [metreeca/self#8] [Q&A]
Hi there, 
i try to use an rd4j-server endpoint in https://demo.metreeca.com/self/ but it's not working.
Did i use the wrong link? -> https://rdf.dev.osalliance.com/rdf4j-server/repositories/kvm_loa

workbench: https://rdf.dev.osalliance.com/rdf4j-workbench/repositories/kvm_loa/query

Thanks Fredy

**Comment by knoan** [comment]
The URL of your endpoint is correct, but metreeca/self (well, actually the browser running it…) requires [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) requests to be supported and that's apparently not your case.

![Screenshot 2021-05-12 at 19 09 55](https://user-images.githubusercontent.com/6105462/118016563-d873ec80-b355-11eb-9fc9-9d927ad923ba.png)

The process of enabling CORS support is server-specific: you may want either to refer to https://enable-cors.org/server.html or to ask for workbench-specific alternatives on the rdf4j mailing list.
